README.md

```diff
@@ -150,10 +150,10 @@ If you want to start the submission service, for example, you start it as follow
 
 #### Debugging
 
-To enable the `DEBUG` log level, you can run the application using the Spring `dev` profile.
+To enable the `DEBUG` log level, you can run the application using the Spring `debug` profile.
```

Comment:
@fredrb / @kilbphilippSAP - I thought, we wanted to name it 'log'!?! or loglevel or debug-log

Reply:
@hilmarf / @fredrb - The task did not specify a name ("Rename dev Spring profile to something like `logging`, `debug` or another more meaningful name for raising the log level."), so I went with `debug`. I can change it to something else, if so desired.